ci/travis_script_python.sh

```diff
@@ -44,19 +41,24 @@ conda activate $CONDA_ENV_DIR
 # python-test fails.
 export ZLIB_HOME=$CONDA_ENV_DIR
 
-python --version
-which python
-
 if [ $ARROW_TRAVIS_PYTHON_JVM == "1" ]; then
   CONDA_JVM_DEPS="jpype1"
 fi
 
-conda install -y -q \
+conda create -y -q -p $CONDA_ENV_DIR \
       --file $TRAVIS_BUILD_DIR/ci/conda_env_python.yml \
+      cmake \
+      curl \
```

Comment:
I'm curious, isn't the system `curl` not enough? Or is not installed on Travis?

Reply:
I'll try to remove it.

Reply:
Let's see...